knowledge-base/daterangepicker-css-customization.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to Customize the DateRangePicker
+description: Learn how to enhance the Blazor DateRangePicker by customizing its styling, disabling weekends, reducing the width of DateInputs, preventing horizontal expansion, hiding Start and End labels, and making the component more compact for better user experience and design consistency in your application.
+type: how-to
+page_title: How to Customize the DateRangePicker
+slug: daterangepicker-kb-css-customization
+tags: daterangepicker, styling
+ticketid: 1633582
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>DateRangePicker for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to customize the Blazor DateRangePicker styling?
+* How to make weekends disabled in the DateRangePicker component?
+* How to reduce the width of the two DateInputs?
+* How to prevent horizontal expansion of the DateRangePicker?
+* How to hide the Start and End labels of the DateRangePicker?
+* How to make the DateRangePicker more compact?
+
+## Solution
+
+1. Use a `Class` parameter and custom CSS to override our theme and hide the labels (**.k-floating-label**). Note that If it is used **display:none**, it violates the accessibility compliance. Instead, use **font-size:0** for example.
```

Comment:
* Don't write in the [first-person plural](https://docs.telerik.com/style-guide/basic-rules-and-guidelines#pronouns).
* [Use `monospace`](https://docs.telerik.com/style-guide/element-formatting#guidelines) (surround the word with backticks) for the code elements.
```suggestion
1. Use a `Class` parameter and custom CSS to override the theme and hide the labels (`.k-floating-label`). Note that using `display:none`, violates the accessibility compliance. Instead, use `font-size:0`, for example.
```